pages/news_page.py

```
@@ -1,8 +1,8 @@
 import allure
-
 from pages.base_page import BasePage
 from pages.locators.eco_news_page_locators import *
 from pages.locators.news_page_locators import *
+from pages.locators.navbar_locators import *
```

Ruff (0.12.2)

`from pages.locators.navbar_locators import *` used; unable to detect undefined names

(F403)